cluster-autoscaler/cloudprovider/gce/gce_manager.go

```diff
@@ -364,6 +367,37 @@ func buildGenericLabels(ref GceRef, machineType string, nodeName string) (map[st
 	return result, nil
 }
 
+func buildReadyConditions() []apiv1.NodeCondition {
```

Comment:
This function doesn't seem GCE-specific at all. Maybe we should move it to some utils file in cloudprovider? WDYT?